src/routes/RootComponent/RootComponent.tsx

```diff
@@ -0,0 +1,32 @@
+import CssBaseline from '@mui/material/CssBaseline';
+import { Suspense, useEffect } from 'react';
+
+import { useAppDispatch } from 'redux/store';
+import { auth } from 'modules/Auth';
+import { useWebsocket } from 'shared/hooks';
+import { alerts } from 'shared/state';
+import { DEFAULT_ROWS_PER_PAGE } from 'shared/state/Alerts/Alerts.const';
+import { Spinner } from 'shared/components';
+import { AppRoutes } from 'routes/index';
+
+export const RootComponent = () => {
```

Comment:
Why not to baselayout?

Reply:
it isn't relating to baselayout, it applies more to routes

Reply:
@alex-i-e but in this case, we also wrap the auth routes and add a check like `if (!isAuthorized) return;`
if we move it to base layout then there will be no need to duplicate the check and we wrap all routes exclude auth